news_layout/index.html

```diff
@@ -2,10 +2,190 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Flexbox Layout</title>
+    <title>Poetry Flex</title>
     <link rel="stylesheet" href="css/main.css">
-  </head>
+    <!--fonts-->
+    <link href="https://fonts.googleapis.com/css?family=Parisienne" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Parisienne|Raleway" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
```

Comment:
You are using 3 different googlefonts but have 4 font-links here. Technically you can write this one!